<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgEncoderSetting</class>
 <widget class="QWidget" name="DlgEncoderSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DlgEncoderSetting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>582</height>
       </rect>
      </property>
      <attribute name="label">
       <string>视频编码</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="3" column="0" colspan="3">
        <widget class="QGroupBox" name="grpOptVideoAdv">
         <property name="title">
          <string>高级设置(请谨慎修改)</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>码率模式：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QComboBox" name="combVideoVBVMode">
            <item>
             <property name="text">
              <string>ABR(平均码率)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>VBR(可变码率)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>CBR(固定码率)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>CQP(恒定质量)</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>最大码率：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>VBV 缓存：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinVBVBitrateMax">
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="singleStep">
             <number>200</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>kbit/s</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>IDR 间隔：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>kbit</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="spinVBVBufSize">
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="singleStep">
             <number>200</number>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="spinVideoIDR">
            <property name="maximum">
             <number>6000</number>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="spinVideoIMin">
            <property name="maximum">
             <number>6000</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>I 帧间隔：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QSpinBox" name="spinVideoRef">
            <property name="maximum">
             <number>16</number>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_25">
            <property name="text">
             <string>Ref 引用：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QSpinBox" name="spinVideoBFrame">
            <property name="maximum">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>B 帧间隔：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="2" rowspan="2">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string/>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="3">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>以下设置保持为 0 则使用默认值</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labVideoEncoderTitle">
         <property name="text">
          <string>编码器：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QComboBox" name="combVideoEncoder"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labVideoPersetTitle">
         <property name="text">
          <string>质量参考：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spinVideoBitrate">
         <property name="minimum">
          <number>200</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="singleStep">
          <number>200</number>
         </property>
         <property name="value">
          <number>800</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labVideoBitrateTitle">
         <property name="text">
          <string>码率：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QComboBox" name="combVideoPerset"/>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>kbit/s</string>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>grpOptVideoAdv</zorder>
      <zorder>spinVideoBitrate</zorder>
      <zorder>labVideoEncoderTitle</zorder>
      <zorder>labVideoBitrateTitle</zorder>
      <zorder>combVideoPerset</zorder>
      <zorder>labVideoPersetTitle</zorder>
      <zorder>label_6</zorder>
      <zorder>combVideoEncoder</zorder>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>582</height>
       </rect>
      </property>
      <attribute name="label">
       <string>音频编码</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>声道：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>kbit/s</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>码率：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinAudioBitrate">
         <property name="minimumSize">
          <size>
           <width>69</width>
           <height>0</height>
          </size>
         </property>
         <property name="minimum">
          <number>16</number>
         </property>
         <property name="maximum">
          <number>256</number>
         </property>
         <property name="singleStep">
          <number>16</number>
         </property>
         <property name="value">
          <number>64</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="radSingleChannel">
         <property name="text">
          <string>单声道</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QRadioButton" name="radDoubleChannel">
         <property name="text">
          <string>立体声</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>采样率：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QComboBox" name="combAudioSamples"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>582</height>
       </rect>
      </property>
      <attribute name="label">
       <string>编码输出</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>标题：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editVideoTitle"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="gboxSaveToFile">
         <property name="title">
          <string>保存到文件</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QLabel" name="labVideoDirTitle">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>视频文件夹：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="combVideoFileFormat">
            <item>
             <property name="text">
              <string>flv</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>mp4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>avi</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labVideoFormatTitle">
            <property name="text">
             <string>文件格式：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="spinVideoSplitSize">
            <property name="maximum">
             <number>8192</number>
            </property>
            <property name="singleStep">
             <number>256</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_36">
            <property name="text">
             <string>文件分卷：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="label_37">
            <property name="text">
             <string>MB</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="QLineEdit" name="editVideoDir"/>
          </item>
          <item row="1" column="1" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="butBrowVideoDir">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>浏览…</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="butViewVideoDir">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>查看…</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QLabel" name="labAcceditLocalDays">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gboxLiveToNet">
         <property name="title">
          <string>直播到网络</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="labAcceditLiveDays">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListView" name="listViewLiveUrls">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkAdaptiveNetwork">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>直播上传时自动适应网络速度</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="butAddLiveUrl">
              <property name="text">
               <string>添加直播目标</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="butDelLiveUrl">
              <property name="text">
               <string>删除选中项</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkOutputVCamera">
         <property name="text">
          <string>输出到虚拟摄像头（暂无功能）</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
